spec/index.html

```diff
@@ -279,8 +279,8 @@ <h3>Effects of Different Entailment Regimes</h3>
         also RDFS-entailed.</p>
         <p>Consider, for example, the following query:</p>
         <pre class="query">SELECT ?prop WHERE { ?prop rdf:type rdf:Property }</pre>
-        <p>Under simple entailment the query has an empty answer when querying the above graph. Under RDF entailment, the <a data-cite="RDF12-SEMANTICS#RDFRules" title=
-        "http://www.w3.org/TR/rdf-mt/#RDFRules">RDF rule</a> <em>rdf1</em> can be used on (5) to derive the triple <code>ex:publishes rdf:type rdf:Property</code> which means that
```

Comment:
Do you have a script that does these conversions?

Reply:
Afraid not.

Reply:
I had presumed that the PR meant that you had completed the conversions you intended. What’s the process you see ahead? We could certainly come ups with some scripts that did much of the changes I applied, but once the output is stable, they wouldn’t be as useful, unless the intent is to always build the resulting spec using these tools, and keep the sources separate.

things I noted:

* change local Alia’s anchors to use `span` instead of `a` due to validation errors.
* Change citations to use `data-cite` instead of `href`. In some cases the references were to versioned documents, in other cases, not.
* Remove rendered bibliography, and replace patterns like `<a href=“#foo”>[foo]</a>` into `[[foo]]`. This often requires using an exported doc short name reference instead of what is in the current doc.
* Use [[[foo]]] to render link using the document name.